specification/monitor/resource-manager/readme.md

```diff
@@ -1539,6 +1501,24 @@ directive:
     from: scheduledQueryRule_API.json
     where: $.definitions.Actions.properties.actionProperties
     reason: "This is a key-value collection which we do not validate and just pass as-is to a service which is several hops down the pipe where they are interpreted. Unknown keys are ignored and there are no invalid values."
+  - suppress: INVALID_TYPE
+    from: metricAlert_API.json
+    reason: "Requires Swagger version update to resolve the types issue"
+  - suppress: GetCollectionOnlyHasValueAndNextLink
+    from: metricAlert_API.json
+    reason: "This suppression is required because the nextLink property is not applicable"
```

Comment:
Can you elaborate why nextLink is not applicable here? Even though you return everything in one page now, you should add it so that you could return multiple pages in the future without breaking the client.

Reply:
I believe these suppressions are being added for issues already in main.  The version of the spec already in main had a number of issues, unrelated to this PR, that were causing check failures.